pkg/collector/corechecks/embed/jmx/loader.go

```diff
@@ -48,28 +44,23 @@ func splitConfig(config integration.Config) []integration.Config {
 
 // Load returns an (empty?) list of checks and nil if it all works out
 func (jl *JMXCheckLoader) Load(config integration.Config) ([]check.Check, error) {
-	var err error
 	checks := []check.Check{}
 
-	if !check.IsJMXConfig(config.Name, config.InitConfig) {
-		return checks, errors.New("check is not a jmx check, or unable to determine if it's so")
```

Comment:
why remove this logic? It's important to report this error if the loader determined that this config has no JMXFetch config (so that it can be reported to the user later)

Reply:
Fixed!